TCAT/Views/SummaryView.swift

```diff
@@ -34,6 +34,9 @@ class SummaryView: UIView {
     /// The primary summary label
     fileprivate var mainLabel = UILabel()
     
+    /// The live indicator
+    fileprivate var liveIndicator = LiveIndicator(size: .small, color: .clear)
```

Comment:
why `fileprivate`? just make it `private` unless other classes in this SummaryView.swift file need to access these vars

Reply:
I thought Swift changed syntax to fileprivate. I should look up the difference

Reply:
TIL https://stackoverflow.com/questions/46588704/what-is-the-difference-between-private-and-fileprivate-in-swift-4